src/bootstrap/bootstrap.ts

```diff
@@ -4,103 +4,161 @@ import path from 'path';
 import fs from 'fs';
 import Logger, { LogLevel, StreamHandler } from '@matrixai/logger';
 
+import { DB } from '@matrixai/db';
+import { Schema } from '@/schema';
 import * as bootstrapErrors from './errors';
-import PolykeyAgent from '../PolykeyAgent';
-
-import * as errors from '../errors';
 import * as utils from '../utils';
-import * as agentUtils from '../agent/utils';
+import { KeyManager, utils as keyUtils } from '../keys';
+import { VaultManager } from '../vaults';
+import { Status } from '../status';
+import { ACL } from '../acl';
+import { GestaltGraph } from '../gestalts';
+import { Sigchain } from '../sigchain';
+import { NodeManager } from '../nodes';
+import { ForwardProxy, ReverseProxy } from '../network';
 
 async function bootstrapPolykeyState(
```

Comment:
This entire function is only used by `CommandBootstrap`.

Now I could put this entire functionality into `CommandBootstrap` instead of having it as a separate function. But I think this could be useful in other scenarios, such as when we want to programmatically bootstrap. So I'm keeping it here.

However the entire file is now `src/bootstrap/utils.ts` so tests will need to be updated.

Reply:
This has been addressed.